snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: guetzli
+version: master
+summary: JPEG encoder for excellent compression density at high visual quality
+description: |
+  Guetzli-generated images are typically 20-30% smaller than images of
+  equivalent quality generated by libjpeg. Guetzli generates only sequential
+  (nonprogressive) JPEGs due to faster decompression speeds they offer.
+
+grade: devel
+confinement: strict
+
+apps:
+  guetzli:
+    command: bin/Release/guetzli
+    plugs: [home]
```

Comment:
The home interface excludes dot files, so the user can't read or write to .bashrc.

This is a transitional interface, which offers a little more access than what we would like, to ease the transition to snaps. In the ubuntu phone we have that filepicker, it's called media hub and some of my teammates are working on snap support there. For CLI applications, we are first collecting feedback from the use of the home and removable-media interfaces, and the classic-confinement, before settling in a final design for file access out of the safe snap space. 

So any feedback you can give us here will be highly appreciated.